schemas/Source.yaml

```diff
@@ -383,6 +392,12 @@ flags:
    - ext_shapeHSM_HsmPsfMoments_flag_no_pixels
    - ext_shapeHSM_HsmPsfMoments_flag_not_contained
    - ext_shapeHSM_HsmPsfMoments_flag_parent_source
+   - ext_shapeHSM_HsmPsfMomentsDebiased_flag
```

Comment:
Posting only because I had to look it up again:
```
'slot_PsfShape'->'ext_shapeHSM_HsmPsfMoments'
```

Reply:
Do you mean make `ixxPSF` be `ext_shapeHSM_HsmPsfMoments_xx` explicitly?

Reply:
Looking back at the Object.yaml, `iDebiasedPSF_flag` looks like a bad naming, since `i` there refers to `i` band and is not the same as the `i` in `ixx` etc. I am more inclined to make the flag names in Object.yaml more meaningful. For e.g., `i_flag` doesn't indicate that it's the object's second-order moments that is flagged. Similarly with `iPSF_flag`.

Reply:
But I'll leave that up to @yalsayyad .  (And maybe a separate ticket).
Other than this, this all looks good to me.

Reply:
Oh, changing all the occurrences of `ixx` -> `Ixx` etc. is going to be a much bigger pain. I'm torn between maintaining consistency between Source and Object tables and making the flag names more meaningful. I'm going to incorporate @yalsayyad 's suggested changes for now, but will wait to hear back before merging.

Reply:
Sorry for confusing everyone. `'slot_PsfShape'->'ext_shapeHSM_HsmPsfMoments'`  wasn't a comment, suggestion or request. It was just informational if anyone was wondering what slot_PsfShape in Source Tables were like I was.